test/other/embind_tsgen_jspi.d.ts

```diff
@@ -1,22 +1,9 @@
 // TypeScript bindings for emscripten-generated code.  Automatically generated at compile time.
-declare namespace RuntimeExports {
-    let HEAPF32: any;
-    let HEAPF64: any;
-    let HEAP_DATA_VIEW: any;
-    let HEAP8: any;
-    let HEAPU8: any;
-    let HEAP16: any;
-    let HEAPU16: any;
-    let HEAP32: any;
-    let HEAPU32: any;
-    let HEAP64: any;
-    let HEAPU64: any;
-}
```

Comment:
I guess this is a side effect of adding `-sSTRICT`?   Is that related to this change somehow?

Reply:
Yeah, strict will remove those definitions. The reporters issue only happened in strict mode, since it turns on erroring on undefined symbols.